packages/story-editor/src/components/library/panes/shopping/productButton.js

```diff
@@ -60,41 +64,86 @@ const StyledButton = styled(Button)`
 const StyledActionsContainer = styled.div`
   margin-right: 10px;
 `;
+
+const DISABLED_PRODUCT_TEXT = __(
+  'Products without images cannot be added.',
+  'web-stories'
+);
+
+const TooltipWrapper = ({ hasImage, children }) => {
+  return !hasImage ? (
+    <Tooltip hasTail title={DISABLED_PRODUCT_TEXT}>
+      {children}
+    </Tooltip>
+  ) : (
+    children
+  );
+};
+
+TooltipWrapper.propTypes = {
+  hasImage: PropTypes.bool,
+  children: PropTypes.node,
+};
+
 function ProductButton({ product, onClick, onFocus, isOnPage }) {
   const ADD_PRODUCT_TEXT = sprintf(
     /* translators: %s: product title. */
     __('Add %s', 'web-stories'),
     product?.productTitle
   );
 
+  const hasImage = Boolean(product?.productImages?.length);
   const REMOVE_PRODUCT_TEXT = sprintf(
     /* translators: %s: product title. */
     __('Remove %s', 'web-stories'),
     product?.productTitle
   );
 
+  let ariaLabel = '';
+  if (!hasImage) {
+    ariaLabel = DISABLED_PRODUCT_TEXT;
+  } else if (isOnPage) {
+    ariaLabel = REMOVE_PRODUCT_TEXT;
+  } else {
+    ariaLabel = ADD_PRODUCT_TEXT;
+  }
+
+  const noop = () => {};
```

Comment:
Nit: can this move outside the component or use `import { noop } from '../../../../utils/noop';`